src/utils/tokens.ts

```diff
@@ -129,3 +130,18 @@ export const getNativeTokenByChainId = (chainId: string): Token => {
     ...selectedNetwork.nativeToken,
   };
 };
+
+export const shouldPreventPaymentsWithTokenInColony = (
+  tokenAddress: string,
+  colony: ColonyFragment,
+  tokenStatesMap: Record<string, boolean>,
```

Comment:
I don't wanna be too nitpicky, but I think `tokenLockedStatesMap` would be a bit more informative.
Or something along the lines of how you named the hook `useTokenLockStates` :D